docs/source/markdown/podmansh.1.md

```diff
@@ -77,11 +85,15 @@ RequiredBy=default.target
 _EOF
 ```
 
-Users inside this container will be allowed to execute containers with SELinux
-separate and able to read and write content in the $HOME/data directory.
+Another example, while runnig as root, create a Quadlet where the users inside this container are allowed to execute containers with SELinux separation and able to read and write content in the $HOME/data directory.
 
 ```
-sudo cat > /etc/containers/systemd/users/podmansh.container << _EOF
+# useradd -s /usr/bin/podmansh fullu
+# grep fullu /etc/passwd
+fullu:x:4010:4010::/home/fullu:/usr/bin/podmansh
+UID=$(id -u fullu)
+# mkdir -p /etc/containers/systemd/users/${UID}
+# cat > /etc/containers/systemd/users/${UID}/podmansh.container << _EOF
```

Comment:
Very minor nit: Maybe I'm missing the reason, but my eye is drawn to the example user names.  Since they're not generic like `exmple1` or `someguy`, it makes me wonder if there's some name significance.  If there isn't, would you consider using more generic example names?

Reply:
There _is_ significance. I like them, they make it immediately obvious what you're doing. I say keep them as they are.

Reply:
I did not look that closely TBH, so if Ed says they're fine, I'm comfortable with that.